Sources/Subprocess/Platforms/Subprocess+Unix.swift

```diff
@@ -118,13 +100,35 @@ extension Execution {
         signal: Signal,
         toProcessGroup shouldSendToProcessGroup: Bool = false
     ) throws {
+        func _kill(_ pid: pid_t, signal: Signal) throws {
+            guard kill(pid, signal.rawValue) == 0 else {
+                throw SubprocessError(
+                    code: .init(.failedToSendSignal(signal.rawValue)),
+                    underlyingError: .init(rawValue: errno)
+                )
+            }
+        }
         let pid = shouldSendToProcessGroup ? -(processIdentifier.value) : processIdentifier.value
-        guard kill(pid, signal.rawValue) == 0 else {
-            throw SubprocessError(
-                code: .init(.failedToSendSignal(signal.rawValue)),
-                underlyingError: .init(rawValue: errno)
-            )
+
+        #if os(Linux)
```

Comment:
I think you may want `|| os(Android)` here too, I'm not sure if os(Linux) applies there.